src/app/projects/projects-list/projects-list.component.css

```diff
@@ -90,9 +90,34 @@ button:focus {
   white-space: nowrap;
 }
 
+.projects_panel_center{
+}
+
 .search_block_main{
+  margin-left: 5px;
+  vertical-align: middle;
+  }
 
-}
+  .search_block_main button{
+    -webkit-appearance: button;
```

Comment:
It's preferable to use some preprocessor for such cases: 
https://github.com/AngularClass/angular-starter/wiki/How-to-include-SCSS-in-components
https://github.com/AngularClass/angular-starter/wiki/How-to-include-PostCSS